src/test/resources/application-test.properties

```diff
@@ -1,5 +1,3 @@
-spring.main.allow-bean-definition-overriding=true
```

Comment:
Moved first to properties to the yaml file.  Don't need to be in here.  Not really related to this piece of work I just noticed it on my code travels. :small_airplane: